image_explorer/image_explorer.py

```diff
@@ -31,6 +31,13 @@ class ImageExplorerBlock(XBlock):  # pylint: disable=no-init
         default="Image Explorer"
     )
 
+    hot_spots = Boolean(
```

Comment:
Can we use a more descriptive name? When this boolean flag is enabled, coordinates given in the hotspots configuration are pointing to the center of where the hotspot images should be located, but when disabled, they point to the top left corner of the image.

Maybe rename to `hotspot_coordinates_centered`?